OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs

```diff
@@ -77,7 +77,13 @@ public void StartGame(Arguments args)
 			// Load a replay directly
 			if (!string.IsNullOrEmpty(Launch.Replay))
 			{
-				var replayMeta = ReplayMetadata.Read(Launch.Replay);
+				ReplayMetadata replayMeta = null;
+				try
+				{
+					replayMeta = ReplayMetadata.Read(Launch.Replay);
+				}
+				catch { }
+
 				if (ReplayUtils.PromptConfirmReplayCompatibility(replayMeta, Game.LoadShellMap))
```

Comment:
> On a related note, our replay-failure error messages need a complete overhaul. Doing that properly is out of scope for this release.

It gives the same "outdated engine" message if you tell it to load a random garbage (like the game binary) too.